tutorials/javaInstallationWindows.md

```diff
@@ -0,0 +1,70 @@
+{% set title="Java 17 Installation Guide for Windows Users" %}
+<frontmatter>
+title: "{{ title }}"
+pageNav: 2
+</frontmatter>
+
+# {{ title }}
+
+This guide explains how to install the **==[Oracle Java 17 JDK](https://www.oracle.com/java/technologies/downloads/#java17)== on Windows**.
```

Comment:
Perhaps "the" is not necessary here.

Reply:
Not too sure about omitting it since its referring to "the" specific JDK and version, but I'm open to change it if you do feel it is unnecessary.

Sorry for the late reply!